src/scripts/pokemons/PokemonList.ts

```diff
@@ -13370,7 +13370,7 @@ const pokemonList: {
             'type': [PokemonType.Grass],
             'eggCycles': 20,
             'levelType': LevelType.mediumslow,
-            'exp': 230,
+            'exp': 236,
```

Comment:
```suggestion
            'exp': 230,
```
Could you change this back to 230:
https://bulbapedia.bulbagarden.net/wiki/Tsareena_(Pok%C3%A9mon)
![image](https://user-images.githubusercontent.com/7288322/76847710-5b5efb80-68a7-11ea-8f74-fbf058a6df27.png)

Reply:
fixed!